Sources/Clappr/Classes/Plugin/Container/SimpleContainerPlugin.swift

```diff
@@ -1,24 +1,41 @@
 open class SimpleContainerPlugin: ContainerPlugin {
+    open var playback: Playback? {
+        return container?.playback
+    }
+
+    open var isNoOpPlayback: Bool {
+        return playback?.pluginName == "NoOp"
```

Comment:
Just a question. Why do we check the name of the plugin instead of its type?
```suggestion
        return playback is NoOpPlayback
```